researchstack-sdk/src/test/java/org/sagebionetworks/bridge/researchstack/BridgeDataProviderTest.java

```diff
@@ -237,6 +240,13 @@ public void testSaveConsent() {
     }
 
     @Test
+    public void saveLocalConsent_scopeOnly() throws Exception {
+        ConsentSignatureBody body = new ConsentSignatureBody();
+        body.scope = SharingScope.ALL_QUALIFIED_RESEARCHERS.toString();
+        dataProvider.saveLocalConsent(context, body);
+    }
```

Comment:
This tests that saveLocalConsent() doesn't crash, but we should also verify that authenticationManager is being called with the correct args.

We should also have a separate test case with _all_ the fields.